remote-api/src/test/resources/alfresco-global.properties

```diff
@@ -1,4 +1,4 @@
 opencmis.tck.readtimeout=10000
 opencmis.tck.connecttimeout=10000
-
+rendition.service.storeRenditionAsProperty.enabled=true

```

Comment:
Should this be false by default ? 

We might need a way to run tests in two ways - ie. configurable, if not already.

Does this relate to the additional test suites added as Travis jobs ?

Reply:
This was accidentally added, the travis job should use `-Drendition.service.storeRenditionAsProperty.enabled` flag to use this feature when testing.